test/formz_test.dart

```diff
@@ -7,24 +7,24 @@ import 'helpers/helpers.dart';
 void main() {
   group('Formz', () {
     group('FormzMixin', () {
-      test('is not valid when default constructor is used', () {
+      test('is not valid when default constructor is used', () async {
         final form = NameInputFormzMixin();
-        expect(form.isValid, isFalse);
-        expect(form.isNotValid, isTrue);
+        expect(await form.isValid, isFalse);
```

Comment:
I think we might want to use `expectLater` over `expect` when awaiting.

```suggestion
        await expectLater(form.isValid, completion(isFalse));
```

Reply:
Done